examples/load-data/README.md

```diff
@@ -0,0 +1,17 @@
+# Connect Saturn Cloud to Your Data
+
+Saturn does not host data, but we know that accessing your data is vital for your data science practice. We have several options for linking your data sources to your Saturn Cloud account. If your data source is not listed here, please contact us at hello@saturncloud.io or on our community Slack and we’ll help you!
+
+## [Load Data From Your Local Filesystem](load-data-local-files.ipynb)
+
+## [Load Data From S3](load-data-s3.ipynb)
+
+## [Load Data From Snowflake](load-data-s3.ipynb)
+
+## [Load Data From Kaggle](load-data-s3.ipynb)
+
+## Next steps
+
+Thanks for trying out this resource! To learn more about how Saturn Cloud works, check out our [Documentation](https://saturncloud.io/docs/), [blog](https://saturncloud.io/s/blog/), or join an [upcoming event](https://saturncloud.io/s/events/).
+
+If you have any questions or suggestions reach out to us at support@saturncloud.io or open an issue on the [examples Github repo](https://github.com/saturncloud/examples).
```

Comment:
1. I'm not a fan of like... the advertising in the docs, ya know?
2. I don't think we should be pushing people to the repo ever.

Thus, I think we can just remove this part entirely

```suggestion
```